src/pages/Cart.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export const Cart = () => {
```

Comment:
writing export statement on bottom will be more readable

Reply:
Hey Mohit, thanks for the review. This just make more sense to me and that's why I have used this way.